crates/ruma-events/src/lib.rs

```diff
@@ -366,6 +368,7 @@ pub enum HasDeserializeFields {
 /// Helper struct to determine if the event has been redacted.
 #[doc(hidden)]
 #[derive(Debug, Deserialize)]
+#[allow(clippy::exhaustive_structs)]
```

Comment:
This lint only triggers on public types, right? I wonder whether it would make sense for it to only trigger on public types that also aren't `doc(hidden)`.

Reply:
As in PR to clippy to make it more restrictive, or at least a clippy issue?

Reply:
Right, for now this is fine but if you could create an issue about this that would be nice.

Reply:
Issue here https://github.com/rust-lang/rust-clippy/issues/7349